assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -79,6 +79,94 @@ WithoutModuleAccess.args = {
 	hasAnalyticsAccess: false,
 	hasAnalytics4Access: false,
 };
+WithoutModuleAccess.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WithoutAnalyticsAccess = Template.bind( null );
+WithoutAnalyticsAccess.storyName = 'Settings w/o Analytics access';
+WithoutAnalyticsAccess.args = {
+	hasAnalyticsAccess: false,
+	hasAnalytics4Access: true,
+};
+WithoutAnalyticsAccess.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+
+export const WithoutAnalytics4Access = Template.bind( null );
+WithoutAnalytics4Access.storyName = 'Settings w/o Analytics-4 access';
+WithoutAnalytics4Access.args = {
+	hasAnalyticsAccess: true,
+	hasAnalytics4Access: false,
+};
+WithoutAnalytics4Access.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).setPropertyID( null );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
```

Comment:
And this one too :)

Reply:
Same as above.

Reply:
As mentioned above, in this case, for GA4 not to have access, it's property ID would've been set (in a real scenario). So setting this to null and having the activate switch would theoretically never happen. So we can remove both these decorators I think.